basics/getting-started/running-pinot-in-docker.md

```diff
@@ -319,6 +319,16 @@ services:
       retries: 5
       start_period: 10s
 
+  pinot-minion:
+    image: ${PINOT_IMAGE:-apachepinot/pinot:1.2.0}
+    command: "StartMinion -zkAddress zookeeper-houseprices:2181"
```

Comment:
Do you mean pinot-zookeeper:2181? Is this docker compose file tested? 
Can you also update relevant services which start up i.e. `docker container ls` output?
Also consider adding how to start minion via docker run in the section above docker compose section?

Reply:
@NihalJain I have made all the suggested changes and tested the docker compose file. Please review